web/client-api/src/main/java/io/deephaven/web/client/api/console/JsVariableChanges.java

```diff
@@ -17,18 +14,37 @@
                                 OTHERWIDGET = "OtherWidget",
                                 PANDAS = "Pandas";
 
-    private JsVariableDefinition[] created;
-    private JsVariableDefinition[] updated;
-    private JsVariableDefinition[] removed;
-
-    private static JsVariableDefinition[] convertDefinitions(VariableDefinition[] definitions) {
-        return Arrays.stream(definitions).map(def -> new JsVariableDefinition(def.getName(), def.getType())).toArray(JsVariableDefinition[]::new);
+    public static String getVariableTypeFromFieldCase(int fieldCase) {
```

Comment:
maybe make an array to hold these, and make the indexes line up with what we expect here? probably easier than if/else/etc

if we have to update, that'll be easier to change too